package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "iggy",
```

Comment:
@spetz @Tchoupinax  we have to change this name before fixing publish since the npm package is already taken https://www.npmjs.com/package/iggy

Reply:
iggy-rs ? iggy-cli ? iggy-client ?